packages/bento-design-system/stories/Components/Table.stories.tsx

```diff
@@ -573,17 +573,34 @@ export const VirtualizedRows = {
   args: {
     stickyHeaders: true,
     height: { custom: 340 },
-    virtualizeRows: true,
+    virtualizeRows: { estimateRowHeight: () => 92 },
```

Comment:
not related to this PR, but I noticed that the table can glitch while scrolling if the estimated height is less than the true one. Since we are defaulting internally to the single line text cell height but with the example data we have multiple lines and thus higher rows, we have to provide the estimates from outside for the examples to work smoothly